android-tests/app/src/androidTest/java/com/azure/autoresttest/BodyStringSyncClientTests.java

```diff
@@ -0,0 +1,135 @@
+package com.azure.autoresttest;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import com.azure.android.core.http.Response;
+import com.azure.android.core.http.exception.HttpResponseException;
+import com.azure.android.core.util.Base64Url;
+import com.azure.androidtest.fixtures.bodystring.StringOperationClient;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(AndroidJUnit4.class)
+public class BodyStringSyncClientTests {
+    private static StringOperationClient client;
+
+    @BeforeClass
+    public static void setup() {
+        client = new StringOperationClient.Builder().host(TestConstants.TestServerRootUrl).build();
+    }
+
+    @Test
+    public void getNull() throws Exception {
+        Assert.assertNull(client.getNullWithRestResponse().getValue());
+    }
+
+    @Test
+    public void putNull() throws Exception {
+        try {
+            Response<Void> putResult = client.putNullWithRestResponse(null);
+            assertEquals(200, putResult.getStatusCode());
+        } catch (Exception ex) {
+            Assert.assertEquals(NullPointerException.class, ex.getClass());
```

Comment:
looks like we are expecting the `putNullWithRestResponse` call to succeeded with 200, if so do we need the try block for NPE?